### PR TITLE
Remove traces of app_mode

### PR DESCRIPTION
**Pull request by MaybeJustJames**

### CODE REVIEW

File: server/scopeserver/dataserver/modules/gserver/GServer.py [hunk 1]
```diff
@@ -578,13 +577,8 @@ def getMyLooms(self, request, context):
...
     def getUUID(self, request, context):
-        if SCope.app_mode:
-            with open(os.path.join(self.dfh.get_config_dir(), "Permanent_Session_IDs.txt"), "r") as fh:
-                newUUID = fh.readline().rstrip("\n")
-                logger.info(f"IP {request.ip} connected to SCope. Running in App mode. Passing UUID {newUUID}.")
-        else:
-            newUUID = str(uuid.uuid4())
-        if newUUID not in self.dfh.get_current_UUIDs().keys():
+        newUUID = str(uuid.uuid4())
+        if newUUID not in self.dfh.get_current_UUIDs():
```
Comment (MaybeJustJames) [inline server/scopeserver/dataserver/modules/gserver/GServer.py:581]: @KrisDavie what should happen if `newUUID` _is_ in the current UUIDs?

Comment (MaybeJustJames) [reply server/scopeserver/dataserver/modules/gserver/GServer.py:581]: Final decision: make this unconditional

Comment (KrisDavie) [reply server/scopeserver/dataserver/modules/gserver/GServer.py:581]: Will you make a new PR for this?

**Review by KrisDavie — APPROVED** [review]
LGTM

**Pull request merged**